engines/savestate.cpp

```diff
@@ -95,7 +95,14 @@ bool SaveStateDescriptor::isAutosave() const {
 
 bool SaveStateDescriptor::hasAutosaveName() const
 {
-	return _description.contains(_("Autosave"));
+	Common::U32String autosave = _("Autosave");
+	if (_description.size() >= autosave.size())
+		return _description.contains(autosave);
```

Comment:
I did contains here so it would be unchanged behavior, but we could also do

`_description.substr(0, autosave.size()) == autosave;`

Reply:
I think it should be safe to replace `contains` with `substr`.